usbl_evologics.orogen

```diff
@@ -35,6 +35,8 @@ task_context "Task" do
    # Set interface with device. SERIAL or ETHERNET.
    # The interface type may be different from the one set by io_port, in case there is an intermediary, i.e. sys_mgm.
    property "interface", "/usbl_evologics/InterfaceType", :SERIAL
+   # Granularity (updateHook() minimum interval in ms, default: 500ms)
+   property "granularity", "uint32_t", 500
```

Comment:
Ah ok.
Anyway, I'd go with base::Time, since it represents a time. And make clear it's default value in description, like you did.